package.json

```diff
@@ -1,36 +1,34 @@
 {
-    "name": "to-do-list",
-    "version": "1.0.0",
-    "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
-    "private": true,
-    "scripts": {
-      "test": "echo \"Error: no test specified\" && exit 1",
-      "build": "webpack"
-    },
-    "repository": {
-      "type": "git",
-      "url": "git+https://github.com/david-lafontant/To-Do-List.git"
-    },
-    "keywords": [],
-    "author": "",
-    "license": "ISC",
-    "bugs": {
-      "url": "https://github.com/david-lafontant/To-Do-List/issues"
-    },
-    "homepage": "https://github.com/david-lafontant/To-Do-List#readme",
-    "devDependencies": {
-      "babel-eslint": "^10.1.0",
-      "css-loader": "^6.5.1",
-      "eslint": "^7.32.0",
-      "eslint-config-airbnb-base": "^14.2.1",
-      "eslint-plugin-import": "^2.25.4",
-      "html-webpack-plugin": "^5.5.0",
-      "style-loader": "^3.3.1",
-      "webpack": "^5.66.0",
-      "webpack-cli": "^4.9.1"
-    },
-    "dependencies": {
-      "lodash": "^4.17.21"
-    }
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```

Comment:
- [ ] Kindly for this project you need to add these 👇  parameters in your script so that when  I run `npm start` your project should be previewed and open in the browser.

```js 

 "scripts": {
    "start": "webpack --config webpack.config.js --watch"
    "build": "webpack serve --open --config webpack.config.js",
  }
```

Reply:
@andersonlebon, please where I had to add it? in the package.json or elsewhere

Reply:
Asked change implemented

Reply:
> * [ ]  Kindly for this project you need to add these 👇  parameters in your script so that when  I run `npm start` your project should be previewed and open in the browser.
> 
> ```js
>  "scripts": {
>     "start": "webpack --config webpack.config.js --watch"
>     "build": "webpack serve --open --config webpack.config.js",
>   }
> ```

@andersonlebon I was unfair to you for this comment. At the time I did not know it was a requirement to have these packages in the project. I only focused on the requirements defined [there](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md#project-requirements). **I apologize.**